### PR TITLE
Abilty to select state.

### PR DESCRIPTION
**Pull request by saipramod**

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -24,6 +26,17 @@ const csvFiles = [
   },
 ];
 
+csv(fs.readFileSync(`./csv/fips.csv`), { columns: true }, (err, output) => {
```
Comment (lzl124631x) [inline src/app.ts:29]: what's this fips for? If it's just about getting the state for each row, we don't really need this. With the first column we already know the state name.

File: src/app.ts [hunk 2]
```diff
@@ -7,8 +7,10 @@ import { mapOwn, toSeries } from "./util";
 import { StackedChartData } from "./models/stackedchart";
 import { Entry, MapDataEntry, MapData } from "./type";
 import { PERCENTILEGROUPS } from "./chartconfiguration/stackedchartconfiguration";
```
Comment (lzl124631x) [inline src/app.ts:9]: let's use `foo-bar` filename and folder name convention for readability

**Comment by lzl124631x** [comment]
I think we can put the mapping between state name and state acronym in the frontend. Don't need that fips API

File: src/app.ts (hunk 1 above)
Comment (saipramod) [reply src/app.ts]: sure moved it

**Pull request merged**